_data/projects/ACMESharp.yml

```diff
@@ -0,0 +1,14 @@
+name: ACMESharp
+desc: .NET Library and PowerShell module for the ACME protocol used by Let's Encrypt
+site: https://github.com/ebekker/ACMESharp
+tags:
+# Note those are tags categorizing your project, not issue labels.
+- ".NET"
+- C#
+- PowerShell
+- Security
+- AWS
+- IIS
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/ebekker/ACMESharp/issues?q=is%3Aopen+is%3Aissue+label%3Aup-for-grabs
```

Comment:
:lipstick: you can simplify this down to `https://github.com/ebekker/ACMESharp/labels/up-for-grabs`